spylib/token.py

```diff
@@ -218,6 +220,10 @@ async def execute_gql(
                 'Flag the access token as invalid.'
             )
             raise ConnectionRefusedError
+
+        if not suppress_errors and 'data' in jsondata and 'errors' in jsondata:
+            raise ShopifyGQLError(jsondata)
```

Comment:
If we move this below the existing block, could we remove the `'data' in jsondata`?

Also, we likely only want to raise if the `errors` array contains at least one item...can't remember if that check achieves this